imap_processing/swapi/swapi_metadata.json

```diff
@@ -0,0 +1,246 @@
+{
+    "global_attrs": {
```

Comment:
I think it is worth still having a "global" global_attrs definition for attributes that are common and should be identical across all data products.

Reply:
This is from that sheet.

I think it might be more descriptive to relabel the nested one with the same name below this in "imap_swapi_l1_sci" to something like "dataset_attrs", which is composed from this **and** the extra fields in that spreadsheet.